views/index.handlebars

```diff
@@ -0,0 +1,55 @@
+<h1 class='mt-8 text-3xl font-medium'>CRUD Teams</h1>
+<div class='flex flex-row w-full items-center justify-between'>
+  <p>There are <strong>{{teams.length}}</strong> teams</p>
+  <a
+    href='/teams/create'
+    class='py-1 px-2 md:py-2 md:px-4 bg-slate-700 hover:bg-slate-800 shadow-sm rounded text-slate-300 transition-all duration-300 ease-in-out'
+    id='button-add'
+  >
+    Add team
+  </a>
+  <button
+    class='py-1 px-2 md:py-2 md:px-4 bg-slate-700 hover:bg-slate-800 shadow-sm rounded text-slate-300 transition-all duration-300 ease-in-out'
+    id='button-reset'
+  >
+    Reset teams
+  </button>
+</div>
+<table
+  class='table-auto border-separate border-spacing-x-1 bg-slate-800 text-slate-300 shadow-sm border-2 border-y-8 md:border-8 border-slate-800 mb-8'
+>
+  <thead class='font-semibold bg-slate-700'>
+    <tr>
+      <td class='p-4 py-2'>Name</td>
+      <td class='p-4 py-2'>Country</td>
+      <td class='p-4 py-2'>Actions</td>
+    </tr>
+  </thead>
+  <tbody>
+    {{#each teams as |team|}}
+      <tr data-team-id='{{team.id}}'>
+        <td class='p-4 py-2'>{{team.name}}</td>
+        <td class='p-4 py-2'>{{team.area.name}}</td>
+        <td class='p-4 py-2 flex flex-row justify-center items-center gap-2'><a
+            href='/teams/{{team.abbreviation}}'
+          ><i
+              class='fa-solid fa-eye hover:text-purple-700 transition-all duration-300 ease-in-out cursor-pointer text-lg'
+            ></i></a>
+          <a href='/teams/{{team.abbreviation}}/edit'>
```

Comment:
Maybe use team.id instead of abbreviation? what could it happen if now,  instead of Premier League clubs, you have more leagues and some of the clubs, have the same short name/ abbreviation between them?

Reply:
You're right, it makes more sense using the unique identifier to handle the teams